docs/includes/virtualenv.txt

```diff
@@ -0,0 +1,8 @@
+.. note:: You can install, upgrade, and remove packages using the program ``pip`` in an isolated environment. You can use ``venv`` or ``virtualenv`` to avoid installing Python             packages globally and to isolate the installation of dependencies of a specific application, so that the dependencies of another application do not interfere. 
```

Comment:
Nit: Stray whitespace between "Python" and "packages". Regarding this text:

> You can install, upgrade, and remove packages using the program ``pip`` in an isolated environment. 

I think the goal of this paragraph is to provide a reusable explanation of how package installation works for people who are new to Python development. With this goal in mind, I would recommend wording like this:

> You can install, upgrade, and remove Python packages using `pip`, the Python package installer. By default, `pip` will attempt to packages system-wide. Use `venv` or `virtualenv` to avoid ...

So that we:
1) introduce `pip` (what is its job in the Python world?)
2) explain its default behavior
3) show the value of virtual environments when using `pip` to manage dependencies.